shared/product-make.include

```diff
@@ -155,6 +155,7 @@ $(OUT)/xccdf-unlinked-withremediations.xml: $(OUT)/xccdf-unlinked-ocilrefs.xml $
 		-stringparam bash_remediations "$(realpath $(OUT)/bash-remediations.xml)" \
 		-stringparam ansible_remediations "$(realpath $(OUT)/ansible-remediations.xml)" \
 		-o $@ $(SHARED)/$(TRANS)/xccdf-addremediations.xslt $(OUT)/xccdf-unlinked-ocilrefs.xml
+	xsltproc -o $@ $(SHARED)/$(TRANS)/xccdf-addcce-remediations.xslt $@
```

Comment:
@mpreisler the reason I did this was because I was having trouble adding this to xccdf-addremediations.xslt. With the following code added to xccdf-addremediations.xslt:
```
         <xsl:if test="@strategy != ''">
              <xsl:attribute name="strategy"><xsl:value-of select="@strategy"/></xsl:attribute>
            </xsl:if>
            <xsl:choose>
              <xsl:when test="contains(., '$CCENUM')">
                <xsl:call-template name="find-and-replace">
                  <xsl:with-param name="text" select="."/>
                  <xsl:with-param name="replace" select="'$CCENUM'"/>
                  <xsl:with-param name="with" select="$ident_cce"/>
                </xsl:call-template>
              </xsl:when>
              <xsl:otherwise>
               <xsl:apply-templates select="node()"/>
              </xsl:otherwise>
           </xsl:choose>   
          </xsl:element>
        </xsl:for-each>
      <xsl:apply-templates select="node()[self::xccdf:check]"/>   
```

It would result in something like:
```
sshd_idle_timeout_value=""

replace_or_append '/etc/sshd/sshd_config' '^ClientAliveInterval' $sshd_idle_timeout_value 'CCE-27433-2' '%s %s'
```

Instead of:
```
sshd_idle_timeout_value="<sub idref="sshd_idle_timeout_value"/>"
<sub idref="function_replace_or_append"/>
replace_or_append '/etc/sshd/sshd_config' '^ClientAliveInterval' $sshd_idle_timeout_value 'CCE-27433-2' '%s %s'
```
in the `<fix />` elements.